src/hammer-vlsi/tech_test.py

```diff
@@ -11,20 +11,24 @@
 import unittest
 
 from hammer_vlsi import HammerVLSISettings
-from test import HammerToolTestHelpers, DummyTool, HasGetTech
-from typing import Any, Dict, List
+import test
+from typing import Any, Dict, List, Optional
 
 from hammer_logging import HammerVLSILogging
 import hammer_tech
 from hammer_tech import LibraryFilter, Stackup, Metal, WidthSpacingTuple
 from hammer_utils import deepdict
 
 
-class HammerTechnologyTest(HasGetTech, unittest.TestCase):
+class HammerTechnologyTest(unittest.TestCase):
```

Comment:
What happened to mixing in `HasGetTech`?

Reply:
When changing imports it is unable to be used in this manner, despite it clearly being in scope based on the reimplemented `def get_tech` below.
I do not know enough about python multiple inheritance to fix is easily.

Reply:
Fixed